dataland-backend/backendOpenApi.json

```diff
@@ -2847,6 +2847,16 @@
         }
       },
       "LksgGeneral": {
+        "required": [
```

Comment:
Please add asterisks for the create-company form as well, and make the "Add Company" button a bar as well

Reply:
Add company is back to just being a button for now since there is no bar in the framework upload form either.

Reply:
ok